server/tests/backends/fixtures/append/domain_name_only.json

```diff
@@ -0,0 +1,154 @@
+{
+  "exclude": [
+    "snowflake",
+    "mysql",
+    "postgres",
+    "pandas"
+  ],
```

Comment:
Should we also run this test for `pandas` & `snowflake` ?

Reply:
let's see :D

Reply:
:heavy_check_mark: for snowflake